src/torrent_info.cpp

```diff
@@ -51,7 +51,9 @@ see LICENSE file.
 
 namespace libtorrent {
 
+#if TORRENT_ABI_VERSION < 4
```

Comment:
removing `from_span` seems unrelated to the main patch, does it interact with the other changes?
CI doesn't test building all deprecation versions, so these kinds of changes need to be manually checked.

Reply:
you are right